src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/AdditionalBookingDetail.kt

```diff
@@ -0,0 +1,109 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToOne
+import org.hibernate.Hibernate
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.common.isEmail
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.common.isUkPhoneNumber
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.common.requireNot
+
+@Entity
+class AdditionalBookingDetail private constructor(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val additionalBookingDetailId: Long = 0,
+
+  @OneToOne
+  @JoinColumn(name = "video_booking_id")
+  val videoBooking: VideoBooking,
```

Comment:
The link to the booking is (intentionally) one sided for now i.e. the booking entity does not have reference to additional details.